calm/flow-example/account-system.json

```diff
@@ -0,0 +1,88 @@
+{
+  "$schema": "https://raw.githubusercontent.com/finos/architecture-as-code/main/calm/draft/2024-04/meta/calm.json",
+  "nodes": [
```

Comment:
Worth adding an id tag with the address to this file (assume it will be from calm.finos.org/flow-example